iperf_automation.py

```diff
@@ -0,0 +1,363 @@
+#!/usr/bin/env python3
+"""
+Automated iperf3 testing script for mobile RF testing
+Runs continuous iperf3 tests while logging GPS coordinates and results
+Now with command-line arguments and detailed logging
+"""
+

```

Comment:
Might want to consider https://iperf3-python.readthedocs.io/en/latest/index.html